go/README.md

```diff
@@ -0,0 +1,7 @@
+This project is a component of the [Operator Framework](https://github.com/operator-framework), an open source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the [introduction blog post](https://coreos.com/blog/introducing-operator-framework).
+
+# Operator SDK Samples - Go
+This directory contains samples of operators powered by Go built using the [operator-sdk][operator_sdk]. To learn more about creating an operator that leverages Go, check out the [user guide][user_guide].
```

Comment:
nitpick, s/go/golang/ at least once for searchability.

Reply:
Done